src/binutils-config

```diff
@@ -318,17 +320,17 @@ uninstall_target() {
 		nm objcopy objdump ranlib readelf size strings strip
 	do
 		x=(
-			"${EROOT}"/usr/bin/${TARGET}-${x}
-			"${EROOT}"/usr/{${HOST}/,}${TARGET}/bin/${x}
-			"${EROOT}"/usr/libexec/gcc/${TARGET}/${x}
+			"${EROOT}"/usr/bin/"${TARGET}"-"${x}"
+			"${EROOT}"/usr/{${HOST}/,}"${TARGET}"/bin/"${x}"
+			"${EROOT}"/usr/libexec/gcc/"${TARGET}"/"${x}"
```

Comment:
Eyebleed city :( if you're quoting nearly an entire string, don't stop quoting just for the staticky-ness of a literal forward slash.